server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/ModularizeJmapDraftAuthenticationStrategyTest.java

```diff
@@ -0,0 +1,260 @@
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, CoreMatchers.equalTo("filter"))
+            .body(ARGUMENTS + ".singleton", hasSize(0));
+    }
+
+    @Test
+    public void getAuthenticationRouteWithDenyAuthenticationStrategyShouldReturnUnauthorizedCode() throws Throwable {
+        setupJamesServerWithCustomAuthenticationStrategy(Optional.of(denyAuthenticationStrategy));
+
+        given()
+        .when()
+            .get("/authentication")
+        .then()
+            .statusCode(401);
+    }
+
+    @Test
+    public void getAuthenticationRouteWhenDefaultAuthenticationStrategiesWithNonAuthenticationShouldReturnUnauthorizedCode() throws Throwable {
+        setupJamesServerWithCustomAuthenticationStrategy(Optional.empty());
+
+        given()
+        .when()
+            .get("/authentication")
+        .then()
+            .statusCode(401);
+    }
+
+    @Test
+    public void getAuthenticationRouteWhenDefaultAuthenticationStrategiesWithValidAccessTokenShouldSucceed() throws Throwable {
+        setupJamesServerWithCustomAuthenticationStrategy(Optional.empty());
+
+        given()
+            .header("Authorization", bobAccessToken.asString())
+        .when()
+            .get("/authentication")
+        .then()
+            .statusCode(200)
+            .body("api", equalTo("/jmap"))
+            .body("eventSource", both(isA(String.class)).and(notNullValue()))
+            .body("upload", equalTo("/upload"))
+            .body("download", equalTo("/download"));
+    }
+
+    @Test
+    public void getFilterWhenDenyAuthenticationStrategyWithNonAuthenticationShouldReturnUnauthorizedCode() throws Throwable {
+        setupJamesServerWithCustomAuthenticationStrategy(Optional.of(denyAuthenticationStrategy));
+
+        String body = "[[" +
+            "  \"getFilter\", " +
+            "  {}, " +
+            "\"#0\"" +
+            "]]";
+
+        given()
+            .body(body)
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(401);
+    }
+
+    @Test
+    public void getFilterWhenDenyAuthenticationStrategyWithValidAccessTokenShouldReturnUnauthorizedCode() throws Throwable {
+        setupJamesServerWithCustomAuthenticationStrategy(Optional.of(denyAuthenticationStrategy));
+
+        String body = "[[" +
+            "  \"getFilter\", " +
+            "  {}, " +
+            "\"#0\"" +
+            "]]";
+
+        given()
+            .header("Authorization", bobAccessToken.asString())
+            .body(body)
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(401);
+    }
+
+    @Test
+    public void getFilterWhenDefaultAuthenticationStrategiesWithValidAccessTokenShouldSucceed() throws Throwable {
+        setupJamesServerWithCustomAuthenticationStrategy(Optional.empty());
+
+        String body = "[[" +
+            "  \"getFilter\", " +
+            "  {}, " +
+            "\"#0\"" +
+            "]]";
```

Comment:
```suggestion
        String body = "[[\"getFilter\",   {}, \"#0\"]]";
```

Would be both acceptable and more readable to me.